lib/SIL/SILVerifier.cpp

```diff
@@ -139,7 +142,10 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
       F.print(llvm::dbgs());
     }
 
-    abort();
+    if (AbortOnFailure)
```

Comment:
Just change this to use exit(1).

Reply:
Terrific, I'll do it early tomorrow and update the pull request.

Reply:
Do you happen to know why the behaviour was to abort (probably historical reasons)?

Reply:
I looked before I wrote that ; ). It is really really old (from revision < 10k IIRC). I think it was just someone working quickly.